src/test/resources/application.properties

```diff
@@ -1 +0,0 @@
-prices.directory=prices-test
```

Comment:
Do not delete this properties file, because it specifies a "prices-test" directory used for testing purposes.

Reply:
Good catch indeed!
Restored the file, thanks!